pkg/manifests/manifests.go

```diff
@@ -1842,6 +1842,18 @@ func (f *Factory) PrometheusAdapterDeployment(apiAuthSecretName string, requesth
 
 	spec.Containers[0].Image = f.config.Images.K8sPrometheusAdapter
 
+	if f.config.ClusterMonitoringConfiguration.K8sPrometheusAdapter == nil {
+		f.config.ClusterMonitoringConfiguration.K8sPrometheusAdapter = &K8sPrometheusAdapter{}
+	}
+
+	if f.config.ClusterMonitoringConfiguration.K8sPrometheusAdapter.Audit == nil {
+		f.config.ClusterMonitoringConfiguration.K8sPrometheusAdapter.Audit = &Audit{}
+	}
+
+	if f.config.ClusterMonitoringConfiguration.K8sPrometheusAdapter.Audit.Profile == "" {
+		f.config.ClusterMonitoringConfiguration.K8sPrometheusAdapter.Audit.Profile = auditv1.LevelMetadata
+	}
+
```

Comment:
do we need this change?

Reply:
This is to keep prometheus adapter tests still running for now. I was trying to keep minimum changes for the PR to backport.

We do have a ticket to remove adapter code it will be removed then.